docs/dml-of-carbondata.md

```diff
@@ -446,6 +446,46 @@ CarbonData DML statements are documented here,which includes:
   ```
   DELETE FROM carbontable WHERE column1 IN (SELECT column11 FROM sourceTable2 WHERE column1 = 'USA')
   ```
+    
+### DELETE STAGE
```

Comment:
Can you please add documentation for flink integration to carbon and can move these new DDL's added for flink and stage to it.

Reply:
No DDL's added  for flink.
Both CarbonInsertFromStageCommand and CarbonDeleteStageFilesCommand are DML.

Reply:
okay.. Can you please add documentation for flink integration to carbon and stage files

Reply:
OK